src/opa.js

```diff
@@ -14,12 +54,19 @@ function stringDecoder(mem) {
   };
 }
 
+/**
+ * Stringifies and loads an object into OPA's Memory
+ * @param {OpaInstance} wasmInstance 
+ * @param {WebAssembly.Memory} memory 
+ * @param {any} value 
+ */
 function _loadJSON(wasmInstance, memory, value) {
   if (value === undefined) {
     throw "unable to load undefined value into memory";
   }
 
   const str = JSON.stringify(value);
+  /** @type {StringAddress} */
```

Comment:
Why do we need these kind of docstrings? Its a local variable in an internal function.. seems like we wouldn't need them?

Reply:
I was running a type check over the module to make sure everything was correct, will remove this.